### PR TITLE
7.7.2

### PR DESCRIPTION
**Pull request by adam-urbanczyk**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by adam-urbanczyk** [comment]
@conda-forge-admin, please rerender

**Comment by whophil** [comment]
The windows builds on this seem to be failing due to "Out of heap space" memory issues.

We used to see this (https://github.com/conda-forge/ocp-feedstock/issues/9) but not recently. Perhaps 7.7.2 requires more memory during compile, for whatever reason. I proposed https://github.com/CadQuery/OCP/issues/90 some time ago. I'm not actually sure it will fix the issue, but it might be worth a shot.

**Comment by adam-urbanczyk** [comment]
Trying to increase the pagefile according to: https://conda-forge.org/docs/maintainer/conda_forge_yml.html

**Comment by adam-urbanczyk** [comment]
@conda-forge-admin, please rerender

**Comment by adam-urbanczyk** [comment]
@whophil seems that it worked. Do you know what needs to be done to get py3.12 in the matrix?

**Comment by adam-urbanczyk** [comment]
@conda-forge-admin, please rerender

**Comment by adam-urbanczyk** [comment]
@conda-forge-admin, please rerender

**Comment by adam-urbanczyk** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -19,3 +19,4 @@ CONDA_PREFIX="${PREFIX}" cmake ${CMAKE_ARGS} -B build -S "${SRC_DIR}/src" \
 
 cmake --build build -j ${CPU_COUNT}
 cmake --install build --prefix "${STDLIB_DIR}"
+cmake -E copy_directory "${SRC_DIR}/src/OCP-stubs" "${SP_DIR}/OCP-stubs"
```
[the remark this answers is not in the document]
Comment (adam-urbanczyk) [reply recipe/build.sh:22]: Yes, mypy will pick them up. For now, I'd prefer to have them in one package.

**Comment by adam-urbanczyk** [comment]
OK, merging.

**Pull request merged**